test/Driver/driver-time-compilation.swift

```diff
@@ -0,0 +1,7 @@
+// RUN: %swiftc_driver -driver-time-compilation %s 2>&1 | %FileCheck %s
```

Comment:
Good idea, but `-parse` appears to trigger a different code path, one that doesn't call `performJobsImpl()`, and so the output for `-driver-time-compilation` is never emitted.